src/build.xml

```diff
@@ -3,7 +3,8 @@
     ToolsVersion="16.0"
     DefaultTargets="Build">
 <PropertyGroup>
-    <Solution>Dynamo.All.sln</Solution>
+    <SolutionName>Dynamo.All</SolutionName>
```

Comment:
SolutionName is part of the msbuild common macros, but only in the VS IDE https://learn.microsoft.com/en-us/cpp/build/reference/common-macros-for-build-commands-and-properties?view=msvc-170
For console builds, we also set the property here